core/src/consensus/sortition/vrf_sortition.rs

```diff
@@ -92,7 +90,7 @@ impl PriorityInfo {
         // CodeChain is using uncompressed form of public keys without prefix, so the prefix is required
         // for public keys to be used in openssl.
 
-        let standard_form_pubkey = [&[SECP256K1_TAG_PUBKEY_UNCOMPRESSED], &signer_public[..]].concat();
+        let standard_form_pubkey = standard_uncompressed_pubkey(signer_public);
```

Comment:
What do you think about moving the above comment("CodeChain is using uncompressed form...") to the `standard_uncompressed_pubkey` function?

Reply:
I moved the comment